.github/workflows/smoketest.yml

```diff
@@ -0,0 +1,81 @@
+name: bridge
+
+on:
+  push:
+    branches:
+      - main
+      - smoketests-on-snowbridge
+  workflow_dispatch:
+  schedule:
+    # Runs at 8:00 AM every day
+    - cron: '0 8 * * *'
+
+jobs:
+  smoketests:
+    runs-on: snowbridge-runner
+    env:
+      CARGO_INCREMENTAL: 0
+      RUST_BACKTRACE: 1
+      RUSTFLAGS: -C debuginfo=1
+    outputs:
+      success: ${{ steps.run_e2e_stack.outcome == 'success' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: snowfork/snowbridge
+          ref: smoketests-on-snowbridge # TODO update to "main" once approved
+      - run: rm -rf polkadot-sdk
+      - uses: actions/checkout@v2
+        with:
+         # repository: paritytech/polkadot-sdk
+         # ref: release-crates-io-v1.7.0 # TODO change back to this branch once Westend code is merged upstream
+          repository: Snowfork/polkadot-sdk
+          ref: snowbridge
+          path: ./polkadot-sdk
```

Comment:
Could we just run against master branch of paritytech/polkadot-sdk?

Reply:
We could, but it would be best to run against the branch that is in production. The reason for this is to catch imminent production changes. For example, the checked hash extension change was merged into master, which we knew, but it was also merged into the production branch, which caught us off-guard.

Reply:
> run against the branch that is in production.

Or run against the next/upcoming release for production? I would assume there is no need to test the current release which should definitely work?

Reply:
The current release changes all the time as changes are merged in (fixes/backported features). So it is definitely needed to run against the current production release branch.